File: src/template/pedro/Multiplicacion.java
package template.pedro;

class Multiplicacion extends Operacion {
    Multiplicacion() {
    }

    Multiplicacion(int redondeo) {
        this.redondeo = redondeo;
    }

    void getOperacion() {
        System.out.println("operación: multiplicación");
    }

    void calcula() {
        resultado = primerValor * segundoValor;
    }
}

File: manuel/template/Multiplicacion.php
<?php 
	require_once "Operacion.php";
	
	class Multiplicacion extends Operacion{
		public function calcula()
		{
			$this->resultado = $this->valor1 * $this->valor2;
		}

		public function getOperacion()
		{
			echo 'Multiplicacion <br/>';
		}
	}
?>